### PR TITLE
Improve store pickup section

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Show relevant information regarding store pickups on `StorePickUpHeader`, such as the name of the store, additional info and receiver's name.
Also, create new `Address` component to better display addresses on `StorePickUpHeader` and `ShippingHeader`.

#### What problem is this solving?
Show information that was missing regarding store pickups and fix layout.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Header/Warnings.js [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
 import PropTypes from 'prop-types'
+import parcelify from '@vtex/delivery-packages'
 
-const Info = ({ split }) => {
-  const orderWasSplit = (split > 1)
+const Info = ({ data }) => {
+  const orderWasSplit = (data.length > 1)
+  const totalParcels = data.reduce((acc, currOrder) => {
+    acc.push(...parcelify(currOrder))
+    return acc
```
Comment (klzns) [inline react/components/Header/Warnings.js]: Usar forma imutavel de manipular arrays

```js
  return [...acc, ...parcelify(currOrder)]
```

File: react/components/Header/Warnings.js [hunk 2]
```diff
@@ -1,26 +1,46 @@
...
+        }
+        {pickup.length > 0 &&
+          <li className={listItem}>
+            <p className="pt2">O prazo de retirada se inicia a partir da confirmação do pagamento</p>
+          </li>
+        }
         {orderWasSplit &&
           <li className={listItem}>
-            <p className="pt2">Sua compra foi dividida em 2 pedidos, pois alguns itens foram vendidos por lojas parceiras. Isso não afeta seus prazos de entrega.</p>
+            <p className="pt2">Sua compra foi dividida em 2 pedidos, pois alguns itens foram vendidos por lojas parceiras. Isso não afeta seus prazos de entrega</p>
```
Comment (klzns) [inline react/components/Header/Warnings.js]:
```suggestion
            <p className="pt2">Sua compra foi dividida em {data.length} pedidos, pois alguns itens foram vendidos por lojas parceiras. Isso não afeta seus prazos de entrega</p>
```

File: react/components/Address.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <div className="c-muted-1 mb3">
+      {
+        pickup && (<p className="c-on-base">{ pickup.pickupFriendlyName }</p>)
+      }
+      <p>
+        {street}, {number}{complement && ` - ${complement}`}
+      </p>
+      <p>
```
Comment (klzns) [inline react/components/Address.js]: Usar o address-form pra renderizar isso

File: react/components/Header/Warnings.js (hunk 1 above)
Comment (victorhmp) [reply react/components/Header/Warnings.js]: Boa! Atualizado.

File: react/components/Header/Warnings.js (hunk 2 above)
Comment (victorhmp) [reply react/components/Header/Warnings.js]: Atualizado

File: react/components/Address.js (hunk 3 above)
Comment (victorhmp) [reply react/components/Address.js]: Atualizado :)

**Pull request merged**